modules/nf-core/purecn/coverage/meta.yml

```diff
@@ -0,0 +1,73 @@
+name: "purecn_coverage"
+description: Calculate intervals coverage for each sample.
+  N.B. the tool can not handle staging files with symlinks, stageInMode should be set to 'link'.
+keywords:
+  - copy number alteration calling
+  - intervals coverage
+  - hybrid capture sequencing
+  - targeted sequencing
+  - DNA sequencing
+tools:
+  - "purecn":
+      description: "Copy number calling and SNV classification using targeted short read sequencing"
+      homepage: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      documentation: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      tool_dev_url: "https://github.com/lima1/PureCN"
+      doi: "10.1186/s13029-016-0060-z"
+      licence: "Artistic-2.0"
+      args_id: "$args"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - intervals:
+      type: file
+      description: Annotated targets optimized for copy number calling
+      pattern: "*.txt"
+
+output:
```

Comment:
Can you use the same names as the output channels here? (e.g. `coverage` should be `txt`)

Reply:
Done